src/SkeletalTrapezoidationEdge.h

```diff
@@ -39,11 +70,53 @@ class SkeletalTrapezoidationEdge
     {
         return is_marked >= 0;
     }
+
+    bool hasTransitions(bool ignore_empty = false) const
+    {
+        return transitions_.use_count() > 0 && (ignore_empty || ! transitions_.lock()->empty());
+    }
+    void initTransitions(std::shared_ptr<std::list<TransitionMiddle>> storage)
```

Comment:
Nitpicking; it's not really initializing them, it's more of a setter (if there is a specific reason for it to be an init, I would like to see that in the documentation)

Reply:
True. I just needed an initializer, but I guess the implementation is broad enough to be a setter. I'll change it.